code/game/machinery/computer/atmos_alert.dm

```diff
@@ -8,6 +8,24 @@
 
 	var/list/priority_alarms = list()
 	var/list/minor_alarms = list()
+	/// Determines if the alerts track via z-level, or only show all station areas
+	var/station_only
```

Comment:
why is this a var if the circuitboard exists as a primary spokesman

Reply:
I'll be honest I was using one of the tram machines heavily for the circuit-to-machine setup and it has the same thing, the variable is on both. I assumed it keeps the setup consistent between board and machine, so roundstart ones still drop the board.

Though thinking about it now... it could probably just be something set on the board on the station_only machine's init, something something `circuit.station_only = TRUE`